### PR TITLE
Clicking an issue number opens a new tab

### PR DESCRIPTION
**Pull request by stowellm**
Addresses #1772

### CODE REVIEW

**Comment by stowellm** [comment]
@jaypoulz basically taught me how to style this `link_to` tag so... you can review it :smile: :goat:

**Comment by kcarcia** [comment]
@stowellm Thanks for taking this one :)

File: app/views/testing/index.html.erb [hunk 1]
```diff
@@ -2,32 +2,42 @@
...
     </div>
     <div id='issues' style="margin-bottom: 20px">
       <% @issues.each do |issue| %>
         <% number = issue['number'] %>
         <div id='issue_<%= number %>'>
           <div id='title_<%= number %>'>
-            <h3><%= link_to(number,issue['html_url']) %>: <%= issue['title'] %></h3>
-            <%= hidden_field("issues[#{number}]", :title, :value => issue['title']) %>
-            <%= hidden_field("issues[#{number}]", :html_url, :value => issue['html_url']) %>
+            <h3><%= link_to(number,issue['html_url'],
```
Comment (jaypoulz) [inline app/views/testing/index.html.erb]: Could you use `"` instead of `'` in .erb files?

File: app/views/testing/index.html.erb [hunk 2]
```diff
@@ -2,32 +2,42 @@
...
-      OS Version 
-      <%= text_field_tag("userinfo[os_version]", "#{@browser.platform}", class: "form-control label-padding") %>
-      <%= link_to("GitHub Personal Access Token", "https://github.com/settings/applications") %>:
-      <%= text_field_tag("userinfo[auth_token]", "", class: "form-control label-padding") %>
+      Broswer Version
+      <%= text_field_tag("userinfo[broswer_version]",
+          "#{@browser.name} #{@browser.full_version}",
+          class: "form-control label-padding") %>
+      OS Version
+      <%= text_field_tag("userinfo[os_version]", "#{@browser.platform}",
```
Comment (jaypoulz) [inline app/views/testing/index.html.erb]: these are strings, right?
why are we embedding strings in strings? can this just be `@browser.platform`

File: app/views/testing/index.html.erb [hunk 3]
```diff
@@ -2,32 +2,42 @@
 <div>
   <%= form_tag("/testing/review", method: :post) do %>
     <div id='userinfo'>
-      Broswer Version 
-      <%= text_field_tag("userinfo[broswer_version]", "#{@browser.name} #{@browser.full_version}", class: "form-control label-padding") %>
-      OS Version 
-      <%= text_field_tag("userinfo[os_version]", "#{@browser.platform}", class: "form-control label-padding") %>
-      <%= link_to("GitHub Personal Access Token", "https://github.com/settings/applications") %>:
-      <%= text_field_tag("userinfo[auth_token]", "", class: "form-control label-padding") %>
+      Broswer Version
```
Comment (jaypoulz) [inline app/views/testing/index.html.erb]: That is not how you spell Browser

**Comment by stowellm** [comment]
@kcarcia @jaypoulz ready for review

**Pull request merged**